src/EnergyPlus/Coils/CoilCoolingDX.hh

```diff
@@ -80,9 +80,10 @@ struct CoilCoolingDX
     static int factory(EnergyPlusData &state, std::string const &coilName);
     static void getInput(EnergyPlusData &state);
     static void clear_state();
+    static void reportAllStandardRatings(EnergyPlusData &state);
```

Comment:
So this is the old coil refactor branch we were working on. And this branch is being compared to develop. So this shows that this was added to this branch, but this is not in develop? No, it's not in develop. So you pulled that function over here to get ready to delete old coils. What confuses me is why in this branch. I guess it doesn't matter as long as it makes it into develop.

Reply:
Yep. The branch we worked on last Feb has never been merged. I've added to it since then and resolved many merge conflicts (any of which could have gone wrong along the way).

Reply:
In fact I'm working on more right now.